GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs

```diff
@@ -314,9 +302,28 @@ private async void GraphQLRequest()
                     subjectId = CurrentSubject.Id
                 }
             };
-            var graphQLResponse = await GraphQLClient.SendQueryAsync(answersRequest);
-            var reductions = graphQLResponse.Data.workflow.subject_reductions;
+
             ResetAnswerCount();
+            GraphQLResponse response = new GraphQLResponse();
+
+            try {
+                response = await GraphQLClient.SendQueryAsync(answersRequest);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("Graph QL Error: {0}", e.Message);
```

Comment:
Will there eventually be any error handling beyond writing to the console?

Reply:
Yes, there probably should be some. It would be good to find an external service, whether that is Rollbar or something else. I'll open an issue about it, because it would be good to automate some of this to keep a keen eye on the table.